### ISSUE TITLE
Swipe button in recyclerview

### ISSUE THREAD

**Issue by AKMohite** [ebanx/swipe-button#53]
I have a recyclerview consisting of cards.
Each card has textview and swipe button
but button gets stuck in between while swiping and does not return to its initial state.

**Comment by hiagodotme** [comment]
Maybe my issue (#59) is related to that too.

**Comment by AKMohite** [comment]
Yes in my recyclerview adapter swipe button gets stuck in between and does not return to original position so I used a normal button for time being.
